src/shared/capture/capturefromfile.cpp

```diff
@@ -119,18 +126,40 @@ bool CaptureFromFile::startCapture()
   
     // Read images to buffer in memory:
     imgs_to_load.sort();
-    imgs_it = imgs_to_load.begin();
-    std::list<std::string>::iterator currentImage = imgs_it;
-    while(currentImage != imgs_to_load.end())
-    {
-      int width(-1);
-      int height(-1);
-      rgba* rgba_img = ImageIO::readRGBA(width, height, currentImage->c_str());
-      fprintf (stderr, "Loaded %s \n", currentImage->c_str());
-      images.push_back(rgba_img);
-      heights.push_back(height);
-      widths.push_back(width);
-      ++currentImage;
+    for (auto &currentImage : imgs_to_load) {
+      int width(v_raw_width->get());
+      int height(v_raw_height->get());
+      if(getFileEnding(currentImage) == "RAW")
+      {
+        std::ifstream file(currentImage, std::ios::binary );
+        if(!file)
+        {
+          std::cout << "Could not read file: " << currentImage << std::endl;
+          continue;
+        }
+        vector<char> buffer((istreambuf_iterator<char>(file)), (istreambuf_iterator<char>()));
+
+        if((int) buffer.size() < width*height)
+        {
+          std::cerr << "Image " << currentImage << " is too small!" << std::endl;
+          continue;
+        }
+
+        RawImage raw_img;
+        raw_img.allocate(ColorFormat::COLOR_RAW8, width, height);
+        memcpy(raw_img.getData(), buffer.data(), static_cast<size_t>(raw_img.getNumBytes()));
+
+        images.push_back(raw_img);
+      }
+      else
+      {
+        cv::Mat srcImg = imread(currentImage, cv::IMREAD_COLOR);
+        RawImage raw_img;
+        raw_img.allocate(ColorFormat::COLOR_RGB8, width, height);
```

Comment:
This seems to assume a specific raw format. If so, `RAW` should be renamed to, for example, `RAWBGR8`.

Reply:
this is actually the 'else' branch. imread from OpenCV is used to read any non-raw image.
I encountered a bug here anyway and fixed it. The conversion was wrong. OpenCV uses BGR, ssl-vision uses RGB format.
PNG files are now read correctly again.

For the raw files, '.raw' is usually the default extension. spinview generates files with this extension, so I don't want to change that.
I could add another parameter for setting the channel size (8,12,16,32), but everything except 8 is rather unusual...